currency-demo/src/App.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+    <v-app>
+        <v-app-bar app color="primary" dark>
+            <div class="d-flex align-center">
+                <h1>USD to INR</h1>
+            </div>
+        </v-app-bar>
+
+        <v-main>
+            <CurrencyConverter
+                v-model="amountInUSD"
+                :exchangeRate="83.92"
+            ></CurrencyConverter>
+            <!-- <CurrencyConverter
```

Comment:
commented code should not be pushed. lets remove this

Reply:
sure, I will follow that